src/components/table/DataTableProperties.js

```diff
@@ -23,7 +27,7 @@ const FORMATTERS = {
   links: ({ links }) =>
     links.map((link, index) => <ResourceList key={index} url={link} />),
   // eslint-disable-next-line react/display-name
-  name: ({ name }) => <a href={name}>{name}</a>,
+  name: ({ name }) => <NameWrapper href={name}>{name}</NameWrapper>,
```

Comment:
Where is this link supposed to point? Going to `name` will just append it to the current route which is not a real route. Why do we need a link around the name anyways?